classes/app.rb

```diff
@@ -0,0 +1,146 @@
+require_relative 'student'
```

Comment:
You made great effort by making this project. However, there are some errors that need to be fixed.